### PR TITLE
challenge phase number added

### PR DESCRIPTION
**Pull request by rakesh-1601**

### CODE REVIEW

File: tests/unit/challenges/test_views.py [hunk 1]
```diff
@@ -1587,9 +1587,12 @@ def test_particular_challenge_phase_does_not_exist(self):
             'error': 'ChallengePhase does not exist'
         }
         response = self.client.get(self.url, {})
-        self.assertEqual(response.data, expected)
-        self.assertEqual(response.status_code, status.HTTP_406_NOT_ACCEPTABLE)
-
+        try:
```
Comment (RishabhJain2018) [inline tests/unit/challenges/test_views.py]: @rakesh-1601 Can you please refer the issue, this PR is solving?

**Comment by RishabhJain2018** [comment]
@rakesh-1601 Any update on this?

File: tests/unit/challenges/test_views.py (hunk 1 above)
Comment (rakesh-1601) [reply tests/unit/challenges/test_views.py]: yeah sure sir..

Comment (rakesh-1601) [reply tests/unit/challenges/test_views.py]: https://github.com/Cloud-CV/EvalAI/pull/1753#issuecomment-415982463

Comment (rakesh-1601) [reply tests/unit/challenges/test_views.py]: now the previous error (assertion error) is solved ...but it giving some kind of warning ..plzz review it

**Comment by rakesh-1601** [comment]
now the warnings are related to some different bug..  https://travis-ci.org/Cloud-CV/EvalAI/jobs/420677713

**Comment by deshraj** [comment]
Closing this due to inactivity.

**Pull request closed without merging**